README.md

```diff
@@ -1,6 +1,6 @@
 ![](amy/static/amy-logo.png)
 
-![](https://travis-ci.org/carpentries/amy.svg?branch=master)
+![](https://travis-ci.com/carpentries/amy.svg?branch=master)
```

Comment:
should the badge show the status of `master` or `develop`?

Reply:
Since `master` contains latest stable version, so I'd say we should show this branch.